Makefile

```diff
@@ -37,10 +61,14 @@ ifdef VT_EXTRA_BUILD_FLAGS
 export EXTRA_BUILD_FLAGS := $(VT_EXTRA_BUILD_FLAGS)
 endif
 
+# This should be the root of the vitess Git directory.
 ifndef VTROOT
 export VTROOT=${PWD}
 endif
 
+# This is where Go will install binaries in response to `go build`.
+export VTROOTBIN=${VTROOT}/bin
+
 # We now have CGO code in the build which throws warnings with newer gcc builds.
 # See: https://github.com/mattn/go-sqlite3/issues/803
 # Work around by dropping optimization level from default -O2.
```

Comment:
Looking at https://github.com/mattn/go-sqlite3/issues/803, I think we can update `go-sqlite3` to the latest version and remove this workaround. 

I'll do that separately though.

Reply:
See https://github.com/vitessio/vitess/pull/10884 for that